client/packages/invoices/src/InboundShipment/api/api.ts

```diff
@@ -119,22 +122,28 @@ const inboundParsers = {
   toDeleteLine: (line: { id: string }): DeleteInboundShipmentLineInput => {
     return { id: line.id };
   },
-  toInsertServiceCharge: (line: DraftInboundLine) => ({
+  toInsertServiceCharge: (
+    line: DraftInboundLine
+  ): InsertInboundShipmentServiceLineInput => ({
     id: line.id,
     invoiceId: line.invoiceId,
     itemId: line.item.id,
     totalBeforeTax: line.totalBeforeTax,
-    tax: line.taxPercentage,
+    taxPercentage: line.taxPercentage,
```

Comment:
Honestly impressed that it didn't throw a type error all the time before, only when tax is set? TIL graphql prunes undefined inputs 👀

Reply:
Nice catch!